src/analytics/serverless.yml

```diff
@@ -1,13 +1,13 @@
 service:
   name: datalake-analytics-api
 
-provider: 
+provider:
   name: aws
   runtime: nodejs8.10
   stage: ${opt:stage, 'dev'}
   deploymentBucket: ${self:service.name}-deploy
   memorySize: 128
-  timeout: 10
+  timeout: 180
```

Comment:
why so much?

Reply:
It can take some time to execute,

Reply:
10seconds isn't enough

Reply:
and why 180?

Reply:
I am saying it because it's an API and I don't want to make kong and client blocking for this long

Reply:
Man, no vortices